modules/uhttp/handler.go

```diff
@@ -59,5 +59,7 @@ type handlerWrapper struct {
 // ServeHTTP calls Handler.ServeHTTP(ctx, w, r) and injects a new service context for use.
 func (h *handlerWrapper) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ctx := fxcontext.New(context.Background(), h.host)
+	stopwatch := h.host.Metrics().SubScope("http").SubScope("handle").Timer("sla").Start()
```

Comment:
Not sure if this it but we need overall latency metrics (including filters and actual execution).
Also need a metric for response status codes.